### PR TITLE
fix verify bugs

### PR DESCRIPTION
**Pull request by grimoire**
Hi
This PR fixes `shape mismatch` when verifying the `whole mode` model in `pytorch2onnx`.

### CODE REVIEW

File: tools/pytorch2onnx.py [hunk 1]
```diff
@@ -96,13 +96,16 @@ def _prepare_input_img(img_path,
     return mm_inputs
 
 
-def _update_input_img(img_list, img_meta_list):
+def _update_input_img(img_list, img_meta_list, update_origin=False):
```
Comment (xvjiarui) [inline tools/pytorch2onnx.py:99]: What does `update_origin` means?

File: tools/pytorch2onnx.py [hunk 2]
```diff
@@ -220,7 +223,7 @@ def pytorch2onnx(model,
 
             # update img_meta
             img_list, img_meta_list = _update_input_img(
-                img_list, img_meta_list)
+                img_list, img_meta_list, test_mode == 'whole')
```
Comment (xvjiarui) [inline tools/pytorch2onnx.py:226]: Why arg is `test_mode == 'whole'`?

Comment (grimoire) [reply tools/pytorch2onnx.py:226]: The input tensor has been interpolated with scale_factor=1.5 to test dynamic shape support.
We only support dynamic shape on these `whole mode` models.

File: tools/pytorch2onnx.py (hunk 1 above)
Comment (grimoire) [reply tools/pytorch2onnx.py:99]: `ori_shape` in img_meta is used to rescale output to origin shape.
`_update_input_img` will update `img_meta_list` according to `img_list`. And this flag is used to control if the segmentator needs to rescale according to the input tensor shape.

**Review by xvjiarui — APPROVED** [review]

**Pull request merged**